### DISCUSSION TITLE
Calculating transport properties of NH3/Water Mixtures

### DISCUSSION THREAD

**Discussion by TimMauN** [usnistgov/REFPROP-wrappers#606] [Q&A]
I just recently bought REFPROP to use in python and I might had not read the documentation as properly as I should have. I wanted it mainly to verify our property calculation tool for NH3/Water mixtures. 
Of course I wanted to calculate following properties: Basic ones like Enthalpie, density, specific heat capacity but also transport properties (viscosity, surface tension, etc). I immediately came across the the TRNPRP error 541. Looking it up in the documentation gave me the information I should have read at the start. "Transport equations are not available for mixtures with water at molar concentrations
greater than 5%" and directly below "Transport equations are not available for the ammonia/water mixture". 

My questions are: 
Is it possible to acquire transport properties of NH3/Water in some way I don't see?
Am I using the wrong functions/FLD data? 
Or do I have to find another use for REFPROP :D

Thanks in advance.

**Comment by ianhbell** [comment]
You are correct - REFPROP does not have a comprehensive model for viscosity for mixtures of ammonia + water. The data situation is tricky, and we don't have yet a great method for mixture modeling for transport properties like we do for thermodynamic properties. Sorry you are on your own and we don't have anything to propose to you.

**Reply by TimMauN** [reply]
Thank you for the fast answer. At least I know now (: . I'll find some other use for REFPROP I am shure. Thank you for your work.
